src/general/StringUtils.cs

```diff
@@ -187,4 +187,37 @@ public static string Unescape(this string input)
 
         return result.ToString();
     }
+
+    /// <summary>
+    ///   Convert a value in a string with up to 3 digits.
+    /// </summary>
+    /// <remarks>
+    ///   If greater than 100, it should use F0, otherwise it should use F1.
+    ///   Above 1000 it should use the number formatting which uses K for
+    ///   thousands, M for millions etc.
+    /// </remarks>
+    /// <param name="value">The value to format</param>
+    /// <returns>The formatted string</returns>
+    public static string ThreeDigitFormat(double value)
+    {
+        if (value >= 1000)
+            return FormatNumber(value);
+
+        if (value >= 100)
+            return value.ToString("F0", CultureInfo.CurrentCulture);
+
+        return value.ToString("F1", CultureInfo.CurrentCulture);
+    }
+
+    /// <summary>
+    ///   Formats two numbers separated by a slash. The numbers will have
+    ///   up to 3 digits each.
+    /// </summary>
+    /// <param name="v1">The first number (numerator)</param>
+    /// <param name="v2">The second number (denominator)</param>
+    /// <returns>The formatted string</returns>
+    public static string SlashSeparatedNumbersFormat(double v1, double v2)
+    {
+        return ThreeDigitFormat(v1) + " / " + ThreeDigitFormat(v2);
```

Comment:
Oh and I think it would be a pretty cheap performance win to use the `$"{variable} / {...}"` string formatting here rather than doing 2 string concatenations.

Reply:
Does it really? I don't use much C# but a couple of places I look suggest otherwise.
[Here one example](https://www.meziantou.net/performance-string-concatenation-vs-string-format-vs-interpolated-string.htm) that I found on the internet. If you want I'll change it anyway, I don't think it should make much of a difference tbh.

Reply:
It seems the advice I've seen that string concatenation will destroy performance doesn't apply in this case. I guess the advice only applies if you iteratively build the string i.e. don't do it in a single go.

I did do a quick test in Thrive and I got:
```
String interpolation: 00:05:03.3813094
String concat: 00:04:35.7005441
```

So a simple 3 value concat on a single line is faster, so I guess maybe it is better to just keep the string concat here?

For interest, here's how the MSIL looks (it seems like the string interpolation causes boxing on the 2 primitive values, whereas the straight concat just calls string conversions on the primitive values and then just makes one call to concat). Interpolated:
```
        IL_0022: ldstr        "{0} / {1}"
        IL_0027: ldloc.3      // baseValue
        IL_0028: box          [mscorlib]System.Single
        IL_002d: ldloc.3      // baseValue
        IL_002e: ldc.r4       5
        IL_0033: add
        IL_0034: box          [mscorlib]System.Single
        IL_0039: call         string [mscorlib]System.String::Format(string, object, object)
        IL_003e: stloc.s      'value'
```
the string concat:
```
       IL_008b: ldloca.s     baseValue_V_7
        IL_008d: call         instance string [mscorlib]System.Single::ToString()
        IL_0092: ldstr        " / "
        IL_0097: ldloc.s      baseValue_V_7
        IL_0099: ldc.r4       5
        IL_009e: add
        IL_009f: stloc.s      V_9
        IL_00a1: ldloca.s     V_9
        IL_00a3: call         instance string [mscorlib]System.Single::ToString()
        IL_00a8: call         string [mscorlib]System.String::Concat(string, string, string)
        IL_00ad: stloc.s      value_V_8
```